roles/rook/README.rst

```diff
@@ -0,0 +1,13 @@
+rook-recovery
```

Comment:
The README needs to explain what all of the **variables** and **tags** are/do.  See https://github.com/ceph/ceph-cm-ansible/tree/master/roles/testnode#usage and below.

It'd be helpful to have example commands under the **Usage** section along with what they do.  You already sort of have that in lines 7-9.  Just provide an example command.

Reply:
Updated the readme file